streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -377,7 +390,17 @@ public void handleAssignment(final Map<TaskId, Set<TopicPartition>> activeTasks,
             throw first.getValue();
         }
 
-        tasks.createTasks(activeTasksToCreate, standbyTasksToCreate);
+        createNewTasks(activeTasksToCreate, standbyTasksToCreate);
+    }
+
+    private void createNewTasks(final Map<TaskId, Set<TopicPartition>> activeTasksToCreate,
+                                final Map<TaskId, Set<TopicPartition>> standbyTasksToCreate) {
+        final Collection<Task> newActiveTasks = activeTasksToCreate.isEmpty() ?
+            Collections.emptySet() : activeTaskCreator.createTasks(mainConsumer, activeTasksToCreate);
+        final Collection<Task> newStandbyTask = standbyTasksToCreate.isEmpty() ?
+            Collections.emptySet() : standbyTaskCreator.createTasks(standbyTasksToCreate);
```

Comment:
Yeah you got me :) this is mainly for unit tests laziness since for many cases we would not need to create active/standby tasks, but still need to mock a no-op function.